Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1795,6 +1796,66 @@ def delete_alert_rule_command(client: AzureSentinelClient, args: Dict[str, Any])
     return CommandResults(readable_output=f'Alert rule {rule_id} was deleted successfully.')
 
 
+def list_subscriptions_command(client: AzureSentinelClient) -> CommandResults:
+
+    full_url = 'https://management.azure.com/subscriptions?api-version=2020-01-01'
+
+    response = client.http_request('GET', full_url=full_url)
+    data_from_response = response.get('value', [])
+
+    readable_output = tableToMarkdown(
```

Comment:
if the list is empty - consider changing the RH to - no subscriptions was found - or something like that.